src/main/java/org/jabref/gui/externalfiletype/EditExternalFileTypeDialog.java

```diff
@@ -0,0 +1,89 @@
+package org.jabref.gui.externalfiletype;
+
+import javax.inject.Inject;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Label;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ToggleGroup;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.desktop.JabRefDesktop;
+import org.jabref.gui.desktop.os.NativeDesktop;
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.gui.util.FileDialogConfiguration;
+import org.jabref.logic.l10n.Localization;
+
+import com.airhacks.afterburner.views.ViewLoader;
+
+public class EditExternalFileTypeDialog extends BaseDialog<Void> {
```

Comment:
I would remove the `Edit` from the name since it is now add/edit combined.

Reply:
I would actually leave it with Edit, for the linked files dialog we also have edit, although one can add files there, too 
And as  we already have another dialog with the name customizeExternalFileTypeDialog, removing the edit would just make things confusing,
I therefore decided to rename it to EditExternalFileTypeEntryDialog